disnake/ext/commands/interaction_bot_base.py

```diff
@@ -1179,7 +1179,7 @@ def decorator(
         ) -> Callable[[ApplicationCommandInteraction], Any]:
```

Comment:
The docstring above (can't comment there directly) also still references `check` instead of `app_check`

Reply:
ccd95a5a